CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusSteppables.py

```diff
@@ -425,14 +440,14 @@ def step(self, mcs):
         for cell in self.cell_list_by_type(self.IMMUNECELL):
 
             cd = self.chemotaxisPlugin.getChemotaxisData(cell, "cytokine")
-            concentration = field[cell.xCOM, cell.yCOM, 0]
-            constant = 50.0 / 10
+            concentration = field[cell.xCOM, cell.yCOM, 1]
+            constant = lamda_chemotaxis
             l = constant / (1.0 + concentration)
             if cell.dict['activated']:
                 cd.setLambda(l)
             else:
                 cd.setLambda(0)
-
+            cd.setLambda(l)
```

Comment:
doesn't this invalidate the else statement? Where you perhaps using this for some testing and forgot to take it out?

Reply:
Yes. This was there for testing purposes. It should be removed.